serviceUsage.html

```diff
@@ -272,6 +272,27 @@ <h2>
             </tr>
```

Comment:
<p>Available REST resources for REMReM Generate Service are described below.</p>
This is for Publish not for generate, in this line change it to publish